controller/model/identity_handlers.go

```diff
@@ -501,6 +502,76 @@ func (handler *IdentityHandler) VisitIdentityAuthenticatorFingerprints(tx *bbolt
 	return stopVisit, err
 }
 
```

Comment:
Allows users to add external ids (i.e. usually emails, employee numbers, etc) that are used to identify the owner of an identity from a separate system.

Necessary when consuming JWTs from external systems where their ID doesn't or can't match Ziti's internal id. Consider a JWT that puts an email in the `sub` field. We need to a way to match that `sub` to an identity for authentication. Enter `externalId`.

I didn't want `externalId` to be on tags because I wanted it to be a separate field that is named and has an index for lookups.

Reply:
Oh, I was just wondering if the spelling was wrong :) You've got Ready instead of Read

Reply:
OH. Yea I'll fix that.